graph/graph_unix.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/docker/docker/pkg/system"
 )
 
-// setupInitLayer populates a directory with mountpoints suitable
+// SetupInitLayer populates a directory with mountpoints suitable
 // for bind-mounting dockerinit into the container. The mountpoint is simply an
 // empty file at /.dockerinit
 //
-// This extra layer is used by all containers as the top-most ro layer. It protects
+// SetupInitLayer this extra layer is used by all containers as the top-most ro layer. It protects
```

Comment:
Agreed, it's weird. If golint is angry on this, I think you can remove the empty comment ``//`` line 22.